スタンダード第27回演習課題

lqmf3 opened the pull request:
・searchStudent（※）,registerStudent,updateStudent,initStudentsCourseのテストを実施 
※searchStudentはgetStudentDetailByIdに名前変更しています

lqmf3 committed:
スタンダード第27回演習課題
・searchStudent（※）,registerStudent,updateStudent,initStudentsCourseのテストを実施
※searchStudentはgetStudentDetailByIdに名前変更しています

TomooAkamatsu commented on src/test/java/raisetech/StudentManagement/service/StudentServiceTest.java:
```diff
@@ -0,0 +1,127 @@
+  }
+
+  @Test
+  void 受講生詳細の登録_受講生とコースがリポジトリに登録されること(){
```
このテストケースに限らず、呼び出されるrepositoryのメソッド全ての挙動をモックで設定しておきましょう！

lqmf3 replied:
ご確認ありがとうございます！
抜けていたregisterStudent,registerStudentCourse,updateStudent,updateStudentCourseのモック設定を追加しました

lqmf3 committed:
スタンダード第27回演習課題_2
registerStudent,registerStudentCourse,updateStudent,updateStudentCourseのモック設定を追加
